src/index.js

```diff
@@ -1,29 +1,75 @@
+/* eslint import/no-unresolved:0 */
 import './style.css';
+import addTodoItem from './modules/addTodoItem.js';
 
-const list = document.getElementById('lists');
-
-const data = [
-  {
-    index: 0,
-    description: 'Go for Shopping',
-    completed: false,
-  },
-  {
-    index: 1,
-    description: 'Finish To Do List Project',
-    completed: false,
-  },
-];
+const getTodoTask = () => {
+  let todos;
+  if (localStorage.getItem('todos') === null) {
+    todos = [];
+  } else {
+    todos = JSON.parse(localStorage.getItem('todos'));
+  }
+  return todos;
+};
```

Comment:
- [ ] **[OPTIONAL]:** There's a one-liner for this logic, which is going to make it cleaner and easier to read using a "Fallback" value, like this: 🤖 

```suggestion
const getTodoTask = () => return JSON.parse(localStorage.getItem('todos')) || []
  


  
  
  
  

```
Basically, the expression to the left of the `||` operator will check if there's a `todos` key in your local storage, if that's true, then returns this value, otherwise it fallback into an empty array.

It would be great if you implemented this! 🤓 🥇